algebras/algebra/src/main/scala/endpoints4s/algebra/StatusCodes.scala

```diff
@@ -47,6 +47,10 @@ trait StatusCodes {
   /** @group operations */
   def IMUsed: StatusCode = unsupportedInterpreter("1.1.0")
 
+  // 3xx Redirection
+  /** @group operations */
+  def NotModified: StatusCode
```

Comment:
We need to provide an implementation to not introduce a backward binary incompatibility:

```suggestion
  def NotModified: StatusCode = unsupportedInterpreter("1.2.0")
```